src/site/content/en/blog/sign-up-form-best-practices/index.md

```diff
@@ -0,0 +1,418 @@
+
+<figure class="w-figure">
+  <img src="images/gmail-activity.png" alt="Gmail account activity page">
+  <figcaption class="w-figcaption"><a href="https://support.google.com/mail/answer/45938?hl=en-GB" 
+    title="Find out how to view Gmail account activity.">Gmail account activity page</a>.</figcaption>
+</figure>
+
+## Make it simple to change or reset passwords {: #password-change}
+
+Make it obvious to users where and how to **update** their account password. On some sites, it's 
+surprisingly difficult.
+
+You should, of course, also make it simple for users to **reset** their password if they forget it. 
+The Open Web Application Security Project provides detailed guidance on [how to handle lost 
+passwords](https://cheatsheetseries.owasp.org/cheatsheets/Forgot_Password_Cheat_Sheet.html).
+
+To keep your business and your users safe, it's especially important to help users change their 
+password if they discover that it's been compromised. To make this easier, you should add a 
+[`/.well-known/change-password`](https://w3c.github.io/webappsec-change-password-url/) URL to your 
+site that redirects to your password management page. This enables password managers to navigate 
+your users directly to the page where they can change their password for your site. This feature is 
+now implemented in Safari, Chrome, and is coming to other browsers. [Help users change passwords 
+easily by adding a well-known URL for changing passwords](/change-password-url) explains how to 
+implement this.
+
+You should also make it simple for users to delete their account if that's what they want.
+
+
+## Offer login via third-party identity providers {: #federated-login}
+
+Many users prefer to log in to websites using an email address and password sign-up form. 
+However, you should also enable users to log in via a third party identity provider, also known as 
+federated login.
+
+<figure class="w-figure">
+  <img src="images/federated-login.png" alt="WordPress login page">
+  <figcaption class="w-figcaption">WordPress login page, with Google and Apple login options.</figcaption>
+</figure>
+
+This approach has several advantages. For users who create an account using federated login, you 
+don't need to ask for, communicate or store passwords. 
+
+You may also be able to access additional verified profile information from federated login, such 
+as an email address—which means the user doesn't have to enter that data and you don't need to do 
+the verification yourself. Federated login can also make it much easier for users when they get a 
+new device.
+
+[Integrating Google Sign-In into your web app](https://developers.google.com/identity/sign-in/web/sign-in) 
+explains how to add federated login to your sign-up options. [Many other](https://en.wikipedia.org/wiki/Federated_identity#Examples) identity platforms are available.
+
+{% Aside %}
+**'First day experience'** is increasingly important since many users now expect to log in from 
```

Comment:
Can you add a link that explains what "first day experience" is? Is it a common UX term?

Reply:
Ah — good point. It's a bit jargony... Can't find a link :/. I've heard the expression used in relation to Android efforts to make moving to a new/additional device as seamless as possible. Maybe needs a little explanation or tweaking?

Something like this: 

**'First day experience'** of a new device is increasingly important since many people now expect to log in from ...

or

**'First day experience'** when you get a new device is increasingly important. User expect to log in from ...

Reply:
I vote for #2, just put it in double quotes.

Ref: https://developers.google.com/style/quotation-marks?hl=en#single-quotation-marks